sample/build.gradle

```diff
@@ -32,8 +26,8 @@ android {
 dependencies {
     compile 'com.android.support:appcompat-v7:22.2.0'
     compile 'com.squareup:otto:+'
-    compile 'com.github.santagroup:santarest:0.0.1'
-    apt 'com.github.santagroup:santarest-compiler:0.0.1'
-//    compile project(":santarest")
-//    apt project(':santarest-compiler')
+//    compile 'com.github.santagroup:santarest:0.0.1'
+//    apt 'com.github.santagroup:santarest-compiler:0.0.1'
+    compile project(":santarest")
+    apt project(':santarest-compiler')
 }
```

Comment:
I specifically did it. I think this commented lines will be needed for release tag. now there is this code at the master.